### ISSUE TITLE
new border segmentation has different opacity when being created

### ISSUE THREAD

**Issue by ngreenwald** [vanvalenlab/deepcell-label#99]
When in border segmentation view, the overlay that is being created has a different transparency setting than the already created segmented view. 

For example, if adding more area to an existing cell, when drawing in pixel editing mode, the parts of the cell that appear underneath the newly added area, before the brush is released, has a different look than the areas that have already been created. 

I'm not sure if this is a side-effect or intended, but it makes difficult to determine where to stop the segmentation contour if the same brightness value appears differently under the brush than outside of it

**Comment by geneva-miller** [comment]
This is a side effect of the way the transparent effect is applied. As the brush draws, it creates new parts of the transparent annotation, but the brush also applies its own transparency to those areas to show where it has been. So, the drawn portion ends up with a different transparency.

For now, this can be circumvented by toggling off highlighting while drawing. Aside from that, there are some image rendering changes and optimizations in the zoom branch that I think would solve this issue once they are incorporated into RGB mode. In that case, I think what this would look like is that the original area of the label still gets the double transparency applied if the brush moves over it, but all other regions where the brush draws get only the normal brush transparency during drawing.

I could throw together a fix for this in the next ~week if you find it to be particularly troublesome, otherwise I will be waiting on this until zoom can be merged in, probably on the order of 2-3 weeks.

**Comment by ngreenwald** [comment]
Great! This is low priority as well, can definitely wait
